ios/Classes/LayerPropertyConverter.swift

```diff
@@ -283,6 +283,39 @@ class LayerPropertyConverter {
         }
     }
 
+    class func addFillExtrusionProperties(
```

Comment:
no need to write this by hand: 
add `fill-extrusion` to `scripts/lib/generate.dart` and run it to generate this code